vllm/spec_decode/draft_model_runner.py

```diff
@@ -0,0 +1,135 @@
+from typing import List, Optional
+
+import torch
+
+from vllm.config import (CacheConfig, DeviceConfig, LoadConfig, LoRAConfig,
+                         ModelConfig, ParallelConfig, SchedulerConfig,
+                         VisionLanguageConfig)
+from vllm.logger import init_logger
+from vllm.sequence import SamplerOutput, SequenceGroupMetadata
+from vllm.worker.model_runner import ModelRunner
+
+logger = init_logger(__name__)
+
+
+class DraftModelRunner(ModelRunner):
```

Comment:
can we add a fat docstring for this class, explaining the motivation and high-level plans?